src/resource/resource-opening-hours/ResourceOpeningHours.tsx

```diff
@@ -63,8 +69,12 @@ const OpeningPeriodsList = ({
             value={language}
           />
           <Button
+            data-cy={addNewOpeningPeriodButtonDataCy}
```

Comment:
I think we have used previously data-test-attribute 🤔

Reply:
Good point! I'll change these data-cy ones to data-test to make them consistent with the ones we have used earlier. Also maybe data-test is better in that sense that it doesn't have to be an attribute just for e2e tests, but also for unit tests.

- [ ] Change data-cy -> data-test